util/network-devp2p/src/discovery.rs

```diff
@@ -389,14 +382,14 @@ impl<'a> Discovery<'a> {
 		node.endpoint.to_rlp_list(&mut rlp);
 		append_expiration(&mut rlp);
 		let old_parity_hash = keccak(rlp.as_raw());
-		let hash = self.send_packet(PACKET_PING, &node.endpoint.udp_address(), &rlp.drain())?;
+		let hash = self.send_packet(PACKET_PING, node.endpoint.udp_address(), &rlp.drain())?;
```

Comment:
Do you think it would make sense to move `rlp` to `send_packet()` to avoid the copying in `assemble_packet()`?

Reply:
Yeah, great catch